screens/Settings/SettingScreenController.ts

```diff
@@ -28,20 +28,18 @@ import {
 } from '../../machines/biometrics';
 import {GlobalContext} from '../../shared/GlobalContext';
 import {useTranslation} from 'react-i18next';
-import {Platform} from 'react-native';
 import {RequestRouteProps, RootRouteProps} from '../../routes';
 import {REQUEST_ROUTES} from '../../routes/routesConstants';
-import {isIOS} from '../../shared/constants';
 
 export function useSettingsScreen(props: RootRouteProps & RequestRouteProps) {
   const {appService} = useContext(GlobalContext);
-  const authService = appService.children.get('auth');
-  const settingsService = appService.children.get('settings');
+  const authService = appService?.children?.get('auth');
+  const settingsService = appService?.children?.get('settings');
```

Comment:
const settingsService = appService?.children?.get('settings');
  Can we have this as
`  const settingsService = appService?.children?.get('settings') || {}` so that we need not have this in every usage?